packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -62,9 +62,20 @@ gmaps.MapOptions _configurationAndStyleToGmapsOptions(
 
   if (configuration.scrollGesturesEnabled == false ||
       configuration.zoomGesturesEnabled == false) {
-    options.gestureHandling = 'none';
+    window.console.warn(
+      '`scrollGesturesEnabled` and `zoomGesturesEnabled` are now ignored on the web.'
+      ' Use `webGestureHandling: WebGestureHandling.none` instead.',
```

Comment:
Won't this warn people who are setting these for mobile? We shouldn't have warnings that are impossible to remove in normal usage.

Reply:
Couldn't we just keep the old behavior if `webGestureHandling` is null?

Reply:
Wouldn't it be possible to want `scrollGesturesEnabled` or `zoomGesturesEnabled` false for mobile but not want `gestureHandling = 'none'`

Reply:
Right you said only if it's null